data/mods/blazemod/items/books/blob_books.json

```diff
@@ -0,0 +1,82 @@
+[
```

Comment:
Probably going to redact these so that I can say the only changes so far have been modernisation, but these roughly encompass the idea of how I want to introduce the recipes.